internal/test/scorecard/tests.go

```diff
@@ -77,6 +78,34 @@ func CryostatCRTest(bundle *apimanifests.Bundle, namespace string, openShiftCert
 	return *r
 }
 
+// CryostatMultiNamespaceTest checks that the operator installs ClusterCryostat in response to a ClusterCryostat CR
+func CryostatMultiNamespaceTest(bundl *apimanifests.Bundle, namespace string, openShiftCertManager bool) scapiv1alpha3.TestResult {
+	tr := newTestResources(CryostatMultiNamespaceTestName)
+	r := tr.TestResult
+
+	err := setupCRTestResources(tr, openShiftCertManager)
+	if err != nil {
+		return fail(*r, fmt.Sprintf("failed to set up %s test: %s", CryostatMultiNamespaceTestName, err.Error()))
+	}
+
+	namespaces := []string{"other-scorecard-namespace"}
+	err = setupTargetNamespace(tr, namespaces[0])
+	if err != nil {
+		return fail(*r, fmt.Sprintf("failed to create an additional namespace for %s test: %s", CryostatMultiNamespaceTestName, err.Error()))
+	}
+	defer cleanupNamespace(r, tr.Client, namespaces[0])
```

Comment:
Would it make sense for the `setupTargetNamespace` to create the namespace list specified in ClusterCryostat, instead of just the first one?